examples/src/main/kotlin/kscience/kmath/operations/ComplexDemo.kt

```diff
@@ -1,18 +1,17 @@
 package kscience.kmath.operations
 
-import kscience.kmath.structures.NDElement
-import kscience.kmath.structures.NDField
-import kscience.kmath.structures.complex
+import kscience.kmath.nd.NDAlgebra
+import kscience.kmath.nd.complex
 
 fun main() {
     // 2d element
-    val element = NDElement.complex(2, 2) { (i,j) ->
+    val element = NDAlgebra.complex(2, 2).produce { (i,j) ->
         Complex(i.toDouble() - j.toDouble(), i.toDouble() + j.toDouble())
     }
     println(element)
 
     // 1d element operation
-    val result = with(NDField.complex(8)) {
+    val result = with(NDAlgebra.complex(8)) {
```

Comment:
The `with` call seems to be redundant. (NDAlgebra.complex(...)) {}

Reply:
I do not like additional braces. With seems idiomatic here.